Dockerfile

```diff
@@ -91,6 +91,12 @@ RUN apt-get update -q && \
     phpdismod pdo_pgsql && \
     phpdismod pgsql && \
     phpdismod xdebug && \
+    # Remove extra extensions installed via packages for other versions of PHP
+    rm -rf /usr/lib/php/20121212 && \
+    rm -rf /usr/lib/php/20131226 && \
+    # rm -rf /usr/lib/php/20151012 && \
```

Comment:
I assume it's commented out as this directory doesn't exist right now?  Is it safer to leave it in, if this directory is not for PHP 7.1?